modules/code-editor/src/views/full/utils/wrapper.ts

```diff
@@ -24,7 +24,7 @@ const wrapper = {
   },
   remove: (content: string, type: string) => {
     if (type === 'bot_config') {
-      return content.replace('bp://types/bot.config.schema.json', '../../bot.config.schema.json')
+      return content.replace('bp://types/bot.config.schema.json', '../bot.config.schema.json')
```

Comment:
The ../ is necessary , because the file is not the same depth as the botpress main config. Ex:
/bots/my_id/bot.config.json
/global/botpress.config.json
/schemas.json

The change currently gives ../bp://types/bot.config.schema.json which doesn't work

Reply:
I am afraid this is not the case, at least with me.  There seems a caching happening somewhere.

To test this, I run `yarn build` and then test, and it is ok.

Anything I am missing?

Reply:
Inside the `bot.config.json` file, the $schema field points to `../../bot.config.schema.json` because when you open the config file in VS Code, you get the correct typings when hovering the bot fields.

When you load that file in the code editor, I replace `../../` with `bp://types/` so the editor can load it correctly. It replaces that with `../..` afterwards so the typing is not messed up in vs code.

I get this at the moment: 
![image](https://user-images.githubusercontent.com/42552874/72925818-52c3cc00-3d21-11ea-8cdd-0b2e3f22ae37.png)

Reply:
You are right, it was because reproducible after deletion of `data` folder.

Modifications done.